inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+is immediately created and you can edit its details in the panels at the
+right (e.g. choose „VERB“ in the PosValue-dropdown to lable it as a
+verb).
 
-Use the *Home* link at the top of the screen to return to the main menu and select *Annotation*
-to open your text file in the annotation editor.
+Congratulations! You have created your first annotation!
 
-To create your first annotation, select *Named entity* from the *Layer* dropdown menu on the right
-side of the screen. Then, use the mouse to select a word in the *Annotation* area. When you release
-the mouse button, the annotation is immediately created and you can edit its details in the right
-sidebar.
+* Let‘s examine the right panel:
 
-Congratulations! You have created your first annotation project.
+* {blank}
+** You find the field „Layer“ on top and „Annotation“ below. In the
+*Layer*-dropdown, you can choose the layer you want to annotate with as
+we yust did. You find several default layers linke Part Of Speech or
+Named Entity already. You can also adjust them and create own layers in
+the project settings (we will deal with the project settings later on).
+INCEpTION supports span layers (annotating a span in the text, e.g. from
+the 51th character („letter“) in the text to the 55th) and relation
+layers (annotating two span annotations). For span layers, the default
+is to always annotate one or more tokens („words“) but you can adjust to
+character level or sentence level. For more details, see the main
+documentation. LINK EINFÜGEN.
+** In the field *Annotation* below, you see the details of a selected
+annotation. It shows the layer the annotation is made on (field „layer“)
+and what has been annotated (field „text“). Below, you can see and
+modify what has been entered for each of the so called *Features*. Each
+layer has different features. E.g. on the Part of Speech-layer, the
+PosValue-feature tells if the token is a noun/verb/adjective/… See the
+main documentation for the different feature types and how to adjust
+features for your individual task. LINK EINFÜGEN
++
+You may enter free text here or work with *Tagsets* to have a well
+defined set of values to enter. You can modify and create tagsets in the
+project settings (section „Tagsets“) which we will deal with later. See
+our main documentation on how to create and modify tagsets. LINK
+EINFÜGEN
+* In the **Annotation Sidebar **at the left (it folds out when clicking
+on the little arrow on top) there are several features you might want to
+check the main documentation for. Especially the *Recommender* section
+(the black speech bubble) is worth a look if you use recommenders
+(recommenders will be explained later on in this guide). E.g. you find
+their measures and learning behaviours. Note the *Search* (the magnifier
+glass): You can create or delete annotations on all or some of the
+search results.
+* To get familiar with INCEpTION, you may want to follow the
```

Comment:
The instructions are also on the project home/dashboard in INCEpTION

Reply:
What do you mean?

Reply:
You say the instructions for the project are on the website, I say they are also on the project home/dashboard in INCEpTION where you have the annotation/kb/.. menu on the left

Reply:
o, sure. makes sense.